jekyll/_cci2/configuration-reference.adoc

```diff
@@ -2653,6 +2653,51 @@ A job can have the keys `requires`, `name`, `context`, `type`, and `filters`.
 
 A job name that exists in your `config.yml`.
 
+'''
+====== `serial-group`
+
+The `serial-group` key is used to add a property to a job to allow a group of jobs to run in series, rather than concurrently, across an organization. Serial groups control the orchestration of jobs across an organization, not just within projects and pipelines.
```

Comment:
We should note that if jobs with the same serial-group key within the same project have lower pipeline numbers, they will be skipped if a job from a higher pipeline number has already been executed.

cc: @nathanfishcircleci @liamclarkedev

Reply:
I agree, that might even deserve its own subsection or somehow highlighted as an important detail.

Something along the lines of:

Serial groups are started in order of pipeline number, if a group is currently waiting or running and another group in the same project attempts to join the queue with a lower pipeline number, it will be `not_run` to maintain order integrity.

If there are no serial groups waiting/running, a pipeline with a lower number can start, such as restoring back to an previous pipeline via a rerun workflow.

Reply:
We can add a caution section